### PR TITLE
Adjusted light theming color for UI 2.0

### LINKED ISSUE #1678: [UI 2.0] Light Theme Colors
**What would you like to be added**:
Light theme colors for our new UI. 

COLORS:
TITLE:#121D59
ICONS: #121D59
TEXT: #000000
BACKGROUND:#E5E5E5 
ScrollToTop background:  #A0D1FB

Please check: #1676

### PR DESCRIPTION
**Pull request by tonyvugithub**
## Issue This PR Addresses
Fixed #1678 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
I made change to `text` property in the theme mostly and keep primary and secondary the same. 
I put the title and icon color as `text.secondary` because I believe icon could be considered a type of text too. 
Please let me know if we need to update this. Don't really know what to do with `primary` and `secondary` now since we have a more minimalistic design.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — APPROVED** [review]

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]
Let's make a new theme using only the colours that we are going to use in UI 2.0

**Comment by tonyvugithub** [comment]
@PedroFonsecaDEV : So I made another push by removing all other unused color and make the dark blue color primary color incase we want to make some background color with it.

**Comment by PedroFonsecaDEV** [comment]
> @PedroFonsecaDEV : So I made another push by removing all other unused color and make the dark blue color primary color incase we want to make some background color with it. The `secondary.main` i think is needed for scrollTo button background

@tonyvugithub Sorry, I forgot the `scrollToTop` background thanks for the catch. The colour is #A0D1FB. 
I will update the issue so we can have it documented. 
Thanks.

File: src/frontend/next/src/theme/index.ts [hunk 1]
```diff
@@ -5,9 +5,7 @@ export const lightTheme: Theme = createMuiTheme({
   palette: {
     type: 'light',
     primary: {
-      main: '#333E64',
-      contrastText: '#E5E5E5',
-      light: '#FFFFFF',
+      main: '#121D59',
     },
     secondary: {
       main: '#0589D6',
```
Comment (PedroFonsecaDEV) [inline src/frontend/next/src/theme/index.ts:11]: Are we using this color on the new UI ?

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]

File: src/frontend/next/src/theme/index.ts (hunk 1 above)
Comment (tonyvugithub) [reply src/frontend/next/src/theme/index.ts:11]: @PedroFonsecaDEV : It's supposed to be for the scrollToTop button. Let me update with the new color.

**Review by PedroFonsecaDEV — APPROVED** [review]
Excellent @tonyvugithub!
Thanks

**Review by chrispinkney — APPROVED** [review]

**Review by izhuravlev — APPROVED** [review]
Looks good to me!

**Comment by PedroFonsecaDEV** [comment]
@tonyvugithub let's keep this commit here, so in the future, you don't need to work on this again.

**Pull request merged**